doc/ref_arch/kubernetes/chapters/chapter04.md

```diff
@@ -27,7 +27,7 @@ Figure 4-1 below shows the architectural components that are described in the su
 <a name="4.2"></a>
 ## 4.2 Host OS
 
-In order for a Host OS to be compliant with this Reference Architecture it must meet the following requirements:
+In order for a Host OS to be compliant with this Reference Architecture, it must meet all of the following requirements:
```

Comment:
I agree with  @gkunz the new text is impossible to satisfy, the old text is better from this perspective.